plugins/pypy/pypy_setup.py

```diff
@@ -335,15 +336,15 @@ def uwsgi_pypy_paste_loader(config):
     """
     global wsgi_application
     c = ffi.string(config)
-    if c.startswith('config:'):
+    if c.startswith(b'config:'):
         c = c[7:]
-    if c[0] != '/':
-        c = os.getcwd() + '/' + c
+    if c[0] != b'/'[0]:
+        c = os.getcwd() + '/' + c.decode()
     try:
-        from logging.config import fileConfig
+        from paste.script.util.logging_config import fileConfig
         fileConfig(c)
     except ImportError:
-        print("PyPy WARNING: unable to load logging.config")
+        print("PyPy WARNING: unable to load paste.script.util.logging_config")
```

Comment:
Is this some old patch that has been forwarded? This have been changed in 71464fbda59c5e3668bcc3fab8c5aa66935cf95c

Reply:
Yes this is, will review this better and submit fixes in the coming days.